{{cookiecutter.project_name}}/README.md

```diff
@@ -0,0 +1,83 @@
+# <center> {{ cookiecutter.friendly_name }}
+
+<div align="center">
+
+[![pypi](https://img.shields.io/pypi/v/{{cookiecutter.project_name}}.svg)](https://pypi.org/project/{{cookiecutter.project_name}}/)
+[![status](https://img.shields.io/pypi/status/{{cookiecutter.project_name}}.svg)](https://pypi.org/project/{{cookiecutter.project_name}}/)
+[![python version](https://img.shields.io/pypi/pyversions/{{cookiecutter.project_name}})](https://pypi.org/project/{{cookiecutter.project_name}})
+[![license](https://img.shields.io/pypi/l/{{cookiecutter.project_name}})](https://opensource.org/licenses/{{cookiecutter.license}})
+
+[![read the docs](https://img.shields.io/readthedocs/{{cookiecutter.project_name}}/latest.svg?label=Read%20the%20Docs
+)](https://{{cookiecutter.project_name}}.readthedocs.io/)
+[![test](https://github.com/{{cookiecutter.github_user}}/{{cookiecutter.project_name}}/workflows/Tests/badge.svg
+)](https://github.com/{{cookiecutter.github_user}}/{{cookiecutter.project_name}}/actions?workflow=Tests)
+[![codecov](https://codecov.io/gh/{{cookiecutter.github_user}}/{{cookiecutter.project_name}}/branch/main/graph/badge.svg
+)](https://codecov.io/gh/{{cookiecutter.github_user}}/{{cookiecutter.project_name}})
+
+[![precommit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white
+)](https://github.com/pre-commit/pre-commit)
+[![black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
+
+</div>
+
+
+## Features
+
+* TODO
+
+
+## Requirements
+
+* TODO
+
+
+## Installation
+
+You can install *{{cookiecutter.friendly_name}}* via [pip][] from [PyPI][]:
+
+```bash
+   $ pip install {{cookiecutter.project_name}}
+```
+
+
+## Usage
+
+Please see the [Command-line Reference][] for details.
```

Comment:
Same thing in this docs with `[]`

Reply:
This file is also outdated. [current content](https://github.com/cauliyang/cookiecutter-hypermodern-python/edit/main/%7B%7Bcookiecutter.project_name%7D%7D/README.md)